docs/release-notes/8.8.asciidoc

```diff
@@ -34,10 +66,8 @@ There are no breaking changes in 8.8.1.
 [discrete]
 [[bug-fixes-8.8.1]]
 ==== Bug fixes 
-* Fixes bugs in the Anomalies table that left {ml} jobs greyed out after they were installed ({pull}158821[#158821]), and that stopped some job counts from being shown ({pull}158739[#158739]).
```

Comment:
@machadoum would you mind confirming that this was only fixed in 8.8.2 and _not_ 8.8.1? I removed this summary after noticing [this comment](https://github.com/elastic/kibana/pull/158739#issuecomment-1583068988) about the fix not making it into the latest BC for 8.8.1.

Reply:
I confirm. It is only fixed on 8.8.2.